.github/workflows/deploy.yml

```diff
@@ -11,6 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - name: Disable BuildKit functionality.
+        run: |
+          sed '/# BEGIN BUILDKIT/,/# END BUILDKIT/d' Dockerfile > fly.Dockerfile
```

Comment:
Instead of doing this in the deploy, I think we should generate and test it during the build phase. 

Altho, we're pulling away from fly.io anyways, so this is fine for now.

Reply:
Main issue is that IDK how we'd persist them during the build workflow. Though, what I can do is check in a `generate-fly-dockerfile.sh` or something.

Reply:
persisting things in builds == `aws s3 cp $file $s3path/$build_number` :sob:

Reply:
Nah, lets not worry about it. This is fine.